src/transforms/stdnames.jl

```diff
@@ -0,0 +1,48 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    StdNames(S)
+
+StdNames transform standardizes the column names.
+"""
+struct StdNames <: Stateless
+  spec::Symbol
+end
+
+StdNames() = StdNames(:upper)
+
+isrevertible(::Type{StdNames}) = true
+
+function apply(transform::StdNames, table)  
+  oldnames = Tables.columnnames(table)
+  spec = transform.spec
+
+  (spec == :camel) && (newnames = map(_camel, oldnames))
+  (spec == :snake) && (newnames = map(_snake, oldnames))
+  (spec == :upper) && (newnames = map(_upper, oldnames))
+
+  names = Dict(zip(oldnames, newnames))
+  table |> Rename(names), oldnames
```

Comment:
Now that is 🥇 👏🏽

Reply:
We can do even better here and save the Rename transform as the cache, so that the revert is just a single call to revert with the cache transform.

Reply:
I did this, but it looks a little bit off. `revert` function of the `Rename` transform takes three arguments `(transform, table, cache)`, but only uses two `(transform, table)`. So we get something like this:
```
revert(cache, newtable, cache)
```
The first cache is cache of StdNames transform. The second cache is supposes to be cache of the Rename transform, but it is not used inside the function anyway, so I just wrote it like this. Wait until I push the commit. It will be cleared then.

Reply:
I think you are not understanding. This is what Júlio is trying to say:
```julia
function apply(transform::StdNames, table)
  # code...
  names = Dict(zip(oldnames, newnames))
  rtrans = Revert(names)
  newtable, cache = apply(rtrans, table)
  newtable, (rtrans, rcache)
end

revert(::StdNames, newtable, cache)
  rtrans, rcache = cache
  revert(rtrans, newtable, rcache)
end
```

Reply:
I think you didn't understand what Júlio meant. This would be the code:
```julia
function apply(transform::StdNames, table)
  # code...
  names = Dict(zip(oldnames, newnames))
  rtrans = Revert(names)
  newtable, cache = apply(rtrans, table)
  newtable, (rtrans, rcache)
end

revert(::StdNames, newtable, cache)
  rtrans, rcache = cache
  revert(rtrans, newtable, rcache)
end
```

Reply:
Oh, I see. Thanks again. You are doing most of the work for me here :))